search-app/src/cmr/search/services/parameters/parameter_validation.clj

```diff
@@ -432,6 +442,23 @@
           []
           years)))))
 
+(defn temporal-facet-month-validation
+  "Validates that the months provided in all temporal-facet parameters are valid."
+  [concept-type params]
+  (when-let [param-values (:temporal-facet params)]
```

Comment:
For simplicity sake, I'd love to see the `when-let` and `when` pulled into a function like `get-temporal-facet-values` (which could take the place of `(vals param-values)`, but I can see that it would take more that just a copy/paste